andela/travel_tool_front
# #160751452 Re-order arrangement of notifications

#### What does this PR do?
Re-orders the arrangement of notifications on the notification pane

#### Description of Task to be completed?
- replace createdAt timestamp with updatedAt

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory and run make start
- Login into the application
- Create a new request and ensure you add yourself as the requesters manager or approve a request
- Open the notification pane.

#### Any background context you want to provide?
Ensure you have as many requests as possible with different dates.
Open pgAdmin or postico... and update the updatedAt section of notifications to different dates

#### What are the relevant pivotal tracker stories?
[#160751452](https://www.pivotaltracker.com/story/show/160751452)

#### Screenshots (if appropriate)
<img width="360" alt="screen shot 2018-09-26 at 3 17 42 pm" src="https://user-images.githubusercontent.com/23608060/46086799-d1960c80-c1a0-11e8-846d-3cb0da33fcf9.png">

#### Questions:
N/A
